<stream>:1:1: Unexpected token
<stream>:1:1: Unexpected token
stack.adb:4:47: info: range check proved (CVC4: 1 VC)
stack.adb:4:47: info: range check proved (CVC4: 4 VC)
stack.adb:5:55: info: range check proved (CVC4: 1 VC)
stack.adb:5:55: info: range check proved (CVC4: 4 VC)
stack.adb:20:22: info: range check proved (CVC4: 1 VC)
stack.adb:21:19: info: index check proved (CVC4: 2 VC)
stack.adb:32:22: info: range check proved (CVC4: 1 VC)
stack.ads:19:14: info: precondition proved (CVC4: 1 VC)
stack.ads:19:14: medium: postcondition might fail, cannot prove Top (S) = X (e.g. when X = 0)
stack.ads:23:14: info: postcondition proved (CVC4: 1 VC)


" , "internal" : true , "results" : [] }
" , "internal" : true , "results" : [] }
Internal error
Internal error
Library file not found: why3.List__my_lists__vectorDefault_Initial_Condition___object.
Library file not found: why3.List__my_lists__vectorDefault_Initial_Condition___object.
None
None
called process error: {"error" : "File /"gnatprove/list/../list.mlw/", line 7141, characters 1-68:
called process error: {"error" : "File /"gnatprove/main/../main.mlw/", line 5741, characters 1-68:
gnatprove: error during flow analysis and proof
